src/cbor.js

```diff
@@ -72,11 +70,15 @@ exports.unmarshal = (input, opts) => {
       if (val instanceof cbor.Tagged) {
```

Comment:
It is just because a tiny change in the version, a module that gets 'reused' and suddenly this verification doesn't work